RadixWallet/Features/TransactionReviewFeature/TransactionReview+Sections.swift

```diff
@@ -934,3 +934,93 @@ extension TransactionReview.ResourceInfo {
 		}
 	}
 }
+
+extension [String: [ResourceIndicator]] {
```

Comment:
nit: I prefer old style: `extension Dictionary where Key == String, Value == ResourceIndicator {` for two reasons:
1. we can search "extension Dictionary" to find ALL dictionary extensions, which is not possible with this, since "extension [" also finds array extension
2. We can use `: Protocol` syntax instead of `==` which is powerful, and only supported by `extension Dictionary` syntax, and thus nice to be consistent.